server/database/seeders/20190530205345-demo-users.js

```diff
@@ -92,6 +92,17 @@ module.exports = {
       status: 'inactive',
       createdAt: new Date(),
       updatedAt: new Date()
+    }, {
+      id: 5445,
```

Comment:
I think we agreed 'id' should be removed since it's auto-incremented

Reply:
Okay

Reply:
done